tests/vmi_multus_test.go

```diff
@@ -698,6 +698,25 @@ var _ = Describe("[Serial]SRIOV", func() {
 			}
 		}
 
+		It("[Serial]should create virt-launcher pod with CAP_SYS_RESOURCE capability", func() {
+			tests.EnableFeatureGate(virtconfig.SRIOVLiveMigrationGate)
+			// Using UpdateKubeVirtConfigValueAndWait expects a test to run in serial mode
```

Comment:
I do not see any other user of this function commenting about this, so there is no reason to comment on it now.
You could add a commit that comments this on the function doctext itself.

Reply:
What I said wasn't accurate, from what I understnad if a test make use of `tests.KubeVirtDefaultConfig` (and not `UpdateKubeVirtConfigValueAndWait`) it is expected to run in serial mode
https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L4291-L4300

Reply:
I think there is only one kubevirt configmap per cluster, so I do not think it matters what value it gets.

Reply:
Can you remove the comment?
 The commit message needs to be updated as well.

Reply:
Done